src/proposal.jl

```diff
@@ -162,8 +162,7 @@ function propose(rng::AbstractRNG, nt::NoUTurnTrajectory{I}, h::Hamiltonian, θ:
     return θ_new, r_new, α / nα
 end
 
-propose(nt::NoUTurnTrajectory{I}, h::Hamiltonian, θ::AbstractVector{T}, r::AbstractVector{T};
-        j_max::Int=10) where {I<:AbstractIntegrator,T<:Real} = propose(GLOBAL_RNG, nt, h, θ, r; j_max=j_max)
+propose(nt::NoUTurnTrajectory{I}, h::Hamiltonian, θ::AbstractVector{T}, r::AbstractVector{T}) where {I<:AbstractIntegrator,T<:Real} = propose(GLOBAL_RNG, nt, h, θ, r)
```

Comment:
I think `transition` is more accurate terminology since this function also performs the `MH` step inside. Often, we would expect `proposal` to be a candidate before the `MH` step.

Reply:
Yes I agree. Those functions were actual proposal before (e.g. MH was outside the one for HMC) but I moved the MH step inside to keep interface more consistent as there is no MH step for the one for NUTS. Let me rename them :)